src/components/_overlays/ContextMenu/ContextMenu.tsx

```diff
@@ -12,12 +14,7 @@ export type MenuItemProps = {
 }
 
 export const ContextMenuItem: React.FC<MenuItemProps> = React.memo(({ icon, onClick, title }) => {
```

Comment:
We don't need ContextMenuItem anymore. Let's remove this and just use `ListItem` inside `ContextMenu`.

Reply:
Good call!